Package.swift

```diff
@@ -73,22 +72,22 @@ let package = Package(
         .testTarget(
             name: "OpenTelemetryInstrumentationSupportTests",
             dependencies: [
-                "OpenTelemetryInstrumentationSupport"
+                "OpenTelemetryInstrumentationSupport",
             ]
         ),
 
         // ==== --------------------------------------------------------------------------------------------------------
         // MARK: Performance / Benchmarks
 
         .target(
-            name: "Benchmarks",
+            name: "TracingBenchmarks",
```

Comment:
Renamed both `Benchmarks` & `SwiftBenchmarkTools` to avoid clashing with the ones defined in `swift-baggage-context`.

Reply:
Good, we should do a similar boring prefix rename in there as well I guess 👍

Reply:
Agreed, let's rename them to `BaggageBenchmarks` & `BaggageBenchmarkTools`. WDYT?

Reply:
Sounds good

Reply:
https://github.com/slashmo/gsoc-swift-baggage-context/pull/21